docs/azure_jumpstart_ag/contoso_supermarket/arc_defender_servers/_index.md

```diff
@@ -0,0 +1,94 @@
+---
+type: docs
+weight: 100
+toc_hide: true
+---
+
+# Infrastructure security with Microsoft Defender for Servers
+
+## Overview
+
+Contoso Supermarket is aware that as they navigate the complex landscape of edge computing, it is essential to protect their servers against threats while maintaining a comprehensive view of their security posture. Microsoft Defender for Servers emerges as a vital component, playing a pivotal role in protecting servers from threats and providing unparalleled visibility into their security landscape.
```

Comment:
```suggestion
Contoso Supermarket are aware that as they navigate the complex landscape of edge computing, it is essential to protect their servers against threats while maintaining a comprehensive view of their security posture. Microsoft Defender for Servers emerges as a vital component, playing a pivotal role in protecting servers from threats and providing unparalleled visibility into their security landscape.
```

I know it might sound like "is" but the context here is Contoso's people, not the actual supermarket.